components/colorslider/index.css

```diff
@@ -45,6 +45,9 @@
 			display: none;
 		}
 	}
+
+	/* Settings for nested Color handle */
+	--mod-colorhandle-hitarea-border-radius: var(--mod-color-slider-handle-hitarea-border-radius, 0px);
```

Comment:
One linting note: custom property definitions generally go at the top of the selector - so on line 26 - rather than at the bottom.  When you turn on VSCode's stylelint plugin, it should report this for you as well.